database/migrations/2020_04_23_093453_create_order_products_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrderProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('order_products', function (Blueprint $table) {
+            $table->bigIncrements('order_product_id');
```

Comment:
The pivot table should be order_table (without _s) and not include order_product_id